docs/reference/esql/functions/examples/ip_prefix.asciidoc

```diff
@@ -0,0 +1,13 @@
+// This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
+
+*Example*
```

Comment:
While you are here, can you perform a magic trick I've just discovered? I need it on a PR that makes new docs and/or generated code. I did it for antlr here: https://github.com/elastic/elasticsearch/pull/109040/files/daaf23fe8c72dc01ed9d6aa230795d40784c09b8#diff-618cd5b83d62060ba3d027e314a21ceaf75d36067ff820db126642944145393e

If you use this `.gitattributes` thing to mark code as generated github will collapse it by default in the diff.

Reply:
Done!